assets/css/style.css

```diff
@@ -562,6 +562,61 @@ a:hover {
   color: #e84c3d;
 }
 
+#sponsors .sponsor-call {
+  display: flex;
+  justify-content: center;
+  padding-left: 5%;
+  padding-right: 5%;
+  padding-top: 1%;
+  padding-bottom: 1.5%;
+  font-size: 16.5px;
+  color: #333333;
+}
+
+#sponsors .sponsor-dropdown {
+  color: #333333;
+  text-align: center;
+  padding-bottom: 1%;
+  font-size: 20px;
+}
+
+#sponsors .sponsor-benefits .color-indicator {
+     font-size: 20px;
+     text-align:center;
```

Comment:
Run Prettier (right click --> format document with --> prettier) on all of the pages you've edited so that our styling is consistent ✨